keyboards/bm60poker/info.json

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
This file is the wrong format, via config goes to <https://github.com/the-via/keyboards>